tower-batch-control/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tower-batch-control"
-version = "0.2.41-beta.16"
+version = "1.0.0-beta.17"
```

Comment:
If we want to bump this to 1.0.0-beta.0, that seems okay too.

```suggestion
version = "0.2.41-beta.17"
```

Reply:
That was a bug. Thanks for noticing it. I'll revert it.

Reply:
I force-pushed a fix.